src/main/java/mate/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package mate;
+
+import java.util.Random;
+
+public class Lottery {
+
+    static final int MAX_NUMBER = 100;
```

Comment:
add access modifier
```suggestion
    static final int MAX_NUMBER = 100;
```

Reply:
Why do I have to add an access modifier to the const?